flock_agent/gui/settings.py

```diff
@@ -27,11 +25,13 @@ def get(self, key):
         return self.settings[key]
 
     def set(self, key, val):
-        self.c.log("Settings", "set", "{} = {}".format(key, val))
+        logger = logging.getLogger("Settings.set")
+        logger.debug(f"{key} = {value}")
```

Comment:
Should be `val`, not `value`. When running this in macOS, I got this stacktrace:

```
Traceback (most recent call last):
  File "/Users/micahlee/code/flock-agent/flock_agent/gui/tabs/settings_tab.py", line 279, in homebrew_update_prompt_toggled
    self.c.gui.settings.set("homebrew_update_prompt", is_checked)
  File "/Users/micahlee/code/flock-agent/flock_agent/gui/settings.py", line 29, in set
    logger.debug(f"{key} = {value}")
NameError: name 'value' is not defined
```